x-pack/plugins/endpoint/public/embeddables/resolver/store/data/selectors.ts

```diff
@@ -405,6 +407,80 @@ export const indexedProcessTree = createSelector(graphableProcesses, function in
   return indexedProcessTreeFactory(graphableProcesses);
 });
 
+/**
+ * Process events that will be graphed.
+ */
+export const relatedEventResults = createSelector(
```

Comment:
I don't think you need `createSelector` here since it's just pulling an immediate property off the passed state. We just use regular functions in our other selector files for these

Reply:
As a matter of style preference there might be something to be said for it being uniform across the selector file, but you're right that it's not needed. I removed it per your suggestion:
https://github.com/elastic/kibana/pull/64121/commits/080e112ff485fd9599c858bc317c8fba49440797